account-integrations/safe/test/hardhat/SafeBlsPlugin.test.ts

```diff
@@ -3,34 +3,31 @@ import { expect } from "chai";
 import { AddressZero } from "@ethersproject/constants";
 import { getBytes, keccak256, solidityPacked } from "ethers";
 import { UserOperationStruct } from "@account-abstraction/contracts";
-import { calculateProxyAddress } from "./utils/calculateProxyAddress";
 import { signer as hubbleBlsSigner } from "@thehubbleproject/bls";
 import { getUserOpHash } from "@account-abstraction/utils";
-
-import { SafeProxyFactory } from "../../typechain-types/lib/safe-contracts/contracts/proxies/SafeProxyFactory";
-import { Safe } from "../../typechain-types/lib/safe-contracts/contracts/Safe";
-import { EntryPoint } from "../../typechain-types/lib/account-abstraction/contracts/core/EntryPoint";
+import { calculateProxyAddress } from "./utils/calculateProxyAddress";
 
 const BLS_PRIVATE_KEY =
   "0xdbe3d601b1b25c42c50015a87855fdce00ea9b3a7e33c92d31c69aeb70708e08";
 const MNEMONIC = "test test test test test test test test test test test junk";
 
-let safeProxyFactory: SafeProxyFactory;
-let safe: Safe;
-let entryPoint: EntryPoint;
```

Comment:
These variables fit much more neatly into the existing pattern with `setupTests`. Declaring uninitialized globals like this means typescript cannot know they have been initialized and puts downward pressure on typing accuracy.